spec/integration/inputs/kafka_spec.rb

```diff
@@ -292,37 +317,58 @@ def delete_remote_schema(schema_registry_client, subject_name)
       expect( subjects ).to be_empty
     end
   end
+end
+
+def cleanup_schema_registry
```

Comment:
`cleanup_schema_registry` reminds me something that's cleaned on the Schema Registry service instead of a "shutdown Schema Registry" action